src/components/modules/api/blocks.ts

```diff
@@ -221,6 +221,7 @@ export default class BlocksAPI extends Module {
   /**
    * Insert new Block and returns it's API
    *
+   * @param {string} id — block id:: optional 
```

Comment:
please add the description explaining the use case for this param

Reply:
@neSpecc I have added the description  of the params and also fixed the lint  error

Reply:
> please add the description explaining the use case for this param

@neSpecc, anything needed to be done?